sql/expression_value.cc

```diff
@@ -561,6 +561,18 @@ EmbeddingValueInfo(std::vector<ssize_t> shape, StorageType storageType)
 {
 }
 
+EmbeddingValueInfo
```

Comment:
Surely this should return a shared pointer?  We basically never use bare objects for EmbeddingValueInfo.